### PR TITLE
Fix: Remove xmlns:wsa's already added elsewhere; select Content-Type HTTP header based on SOAP version

### PR DESCRIPTION
**Pull request by armstrtw**
Couple of changes I needed for multipart support to work with a server using Axis2.

I'm having a little trouble running the test suite on my system, but I'll work on that.

### CODE REVIEW

**Comment by pcai** [comment]
Thanks for contributing this back. Are you still having trouble running tests? If so, please share the output of `bundle exec rake --trace`

File: lib/savon/operation.rb [hunk 1]
```diff
@@ -11,6 +11,11 @@
 module Savon
   class Operation
 
+    SOAP_REQUEST_TYPE = {
```
Comment (olleolleolle) [inline lib/savon/operation.rb:14]: We _could_ call it content types in the name, I guess.

File: lib/savon/header.rb [hunk 2]
```diff
@@ -62,13 +62,8 @@ def build_wsa_header
...
          'wsa:To' => @globals[:endpoint],
-         'wsa:MessageID' => "urn:uuid:#{SecureRandom.uuid}",
-         attributes!: {
-          'wsa:MessageID' => {
-            "xmlns:wsa" => "http://schemas.xmlsoap.org/ws/2004/08/addressing"
-          }
-         }
-       })
+         'wsa:MessageID' => "urn:uuid:#{SecureRandom.uuid}"
+                      })
```
Comment (olleolleolle) [inline lib/savon/header.rb:66]: The whitespace here is wonky.

```suggestion
       })
```

**Comment by yllihv27** [comment]
xmlns:wsa attribute is not needed on wsa:MessageID. 

In what savon version is this applied?  I need this version, dont know how to remove the attribute. I am using 2.12.1

**Comment by Piioo** [comment]
Im not sure if because of this, but now the https://github.com/savonrb/savon-multipart gem is not working anymore

**Pull request merged**